easybuild/easyconfigs/y/yank/yank-0.25.2-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,108 @@
+easyblock = 'PythonBundle'
+
+name = 'yank'
+version = '0.25.2'
+versionsuffix = "-Python-%(pyver)s"
+
+homepage = 'https://getyank.org/'
+description = "A GPU-accelerated Python framework for exploring algorithms for alchemical free energy calculations"
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('matplotlib', '3.2.1', versionsuffix),
+    ('OpenMM', '7.4.2', versionsuffix),
+    ('MDTraj', '1.9.4', versionsuffix),
+    ('OpenMMTools', '0.20.0', versionsuffix),
+    ('pymbar', '3.0.3', versionsuffix),
+    ('PyYAML', '5.3'),
+    ('IPython', '7.15.0', versionsuffix),
+    ('PyTables', '3.6.1', versionsuffix),
+    ('packmol', 'v20.2.2'),
+    ('libxslt', '1.1.34'),
+    ('libxml2', '2.9.10'),
+    ('ParmEd', '3.2.0', versionsuffix),
+    ('AmberTools', '20', versionsuffix),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('docopt', '0.6.2', {
+        'checksums': ['49b3a825280bd66b3aa83585ef59c4a8c82f2c8a522dbe754a8bc8d08c85c491'],
+    }),
+    ('six', '1.15.0', {
+        'checksums': ['30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259'],
+    }),
+    ('lxml', '4.6.1', {
+        'checksums': ['c152b2e93b639d1f36ec5a8ca24cde4a8eefb2b6b83668fcd8e83a67badcb367'],
+    }),
+    ('numpydoc', '1.1.0', {
+        'checksums': ['c36fd6cb7ffdc9b4e165a43f67bf6271a7b024d0bb6b00ac468c9e2bfc76448e'],
+    }),
+    ('openmoltools', '0.8.7', {
+        'modulename': False,
+        'source_tmpl': '0.8.7.tar.gz',
+        'source_urls': ['https://github.com/choderalab/openmoltools/archive'],
+        'checksums': ['dae37dc1136cbc6175b1935e2b4352cc7cc9c9e4d441968178c9e90ec674a1df'],
+    }),
+    ('mpiplus', 'v0.0.1', {
+        'source_tmpl': 'v0.0.1.tar.gz',
```

Comment:
please use `v%(version)s.tar.gz` (don't hardcode version numbers in multiple places)